tests/unit/test_changelog_section.py

```
@@ -0,0 +1,80 @@
+
+## 2.5.0
+
+- older stuff
+"""
+
+
+class TestExtractSection:
+    def test_extracts_named_version(self):
+        body = extract_section(SAMPLE, "2.6.0")
+        assert body is not None
+        assert "thing one" in body and "thing two" in body
```

Ruff (0.15.17)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)